.compatibility_tests/compatibility_test_1_4/tests/integration/compatibility_1_4.rs

```diff
@@ -0,0 +1,142 @@
+#[test_case(test_roqoqo_1_4::operations::MolmerSorensenXX::new(0,1).into(); "MolmerSorensenXX")]
+#[test_case(test_roqoqo_1_4::operations::VariableMSXX::new(0,1, 0.1.into()).into(); "VariableMSXX")]
+#[test_case(test_roqoqo_1_4::operations::GivensRotation::new(0,1, 0.1.into(), 0.1.into()).into(); "GivensRotation")]
+#[test_case(test_roqoqo_1_4::operations::GivensRotationLittleEndian::new(0,1, 0.1.into(), 0.1.into()).into(); "GivensRotationLittleEndian")]
+#[test_case(test_roqoqo_1_4::operations::Qsim::new(0,1, 0.1.into(), 0.1.into(), 0.1.into()).into(); "Qsim")]
+#[test_case(test_roqoqo_1_4::operations::Fsim::new(0,1, 0.1.into(), 0.1.into(), 0.1.into()).into(); "Fsim")]
+#[test_case(test_roqoqo_1_4::operations::SpinInteraction::new(0,1, 0.1.into(), 0.1.into(), 0.1.into()).into(); "SpinInteraction")]
+#[test_case(test_roqoqo_1_4::operations::Bogoliubov::new(0,1, 0.1.into(), 0.1.into()).into(); "Bogoliubov")]
+#[test_case(test_roqoqo_1_4::operations::PMInteraction::new(0,1, 0.1.into()).into(); "PMInteraction")]
+#[test_case(test_roqoqo_1_4::operations::ComplexPMInteraction::new(0,1, 0.1.into(), 0.1.into()).into(); "ComplexPMInteraction")]
+#[test_case(test_roqoqo_1_4::operations::PhaseShiftedControlledZ::new(0,1, 1.0.into()).into(); "PhaseShiftedControlledZ")]
+#[test_case(test_roqoqo_1_4::operations::PhaseShiftState1::new(0, 1.0.into()).into(); "PhaseShiftState1")]
+#[test_case(test_roqoqo_1_4::operations::PhaseShiftState0::new(0, 1.0.into()).into(); "PhaseShiftState0")]
+#[test_case(test_roqoqo_1_4::operations::MultiQubitMS::new(vec![0,2,3], 1.0.into()).into(); "MultiQubitMS")]
+#[test_case(test_roqoqo_1_4::operations::MultiQubitZZ::new(vec![0,2,3], 1.0.into()).into(); "MultiQubitZZ")]
+// 1.1 and 1.2
+#[test_case(test_roqoqo_1_4::operations::InputBit::new("input".to_string(), 1, true).into(); "InputBit")]
+#[test_case(test_roqoqo_1_4::operations::PragmaLoop::new(2.0.into(), test_roqoqo_1_4::Circuit::new()).into(); "PragmaLoop")]
+#[test_case(test_roqoqo_1_4::operations::PhaseShiftedControlledPhase::new(0,1, 1.0.into(), 1.0.into()).into(); "PhaseShiftedControlledPhase")]
+// 1.3
+#[test_case(test_roqoqo_1_4::operations::ControlledRotateX::new(0,1, 1.0.into()).into(); "ControlledRotateX")]
+#[test_case(test_roqoqo_1_4::operations::ControlledRotateXY::new(0,1, 1.0.into(), 1.0.into()).into(); "ControlledRotateXY")]
+#[test_case(test_roqoqo_1_4::operations::ControlledControlledPauliZ::new(0,1,2).into(); "ControlledControlledPauliZ")]
+#[test_case(test_roqoqo_1_4::operations::ControlledControlledPhaseShift::new(0,1,2, 1.0.into()).into(); "ControlledControlledPhaseShift")]
+#[test_case(test_roqoqo_1_4::operations::Toffoli::new(0,1,2).into(); "Toffoli")]
+// 1.4 - add this into the next compatibility crate
+#[test_case(test_roqoqo_1_4::operations::GPi::new(0, 0.1.into()).into(); "GPi")]
+#[test_case(test_roqoqo_1_4::operations::GPi2::new(0, 0.1.into()).into(); "GPi2")]
+
```

Comment:
I would suggest adding in the operations (commented out) added in version 1.5 here, with a comment above to uncomment them in the next compatibility crate, so it's easier to keep track of

Reply:
Added it.